rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-download-from-google-drive.adoc

```diff
@@ -0,0 +1,168 @@
+** *Google API Service Account Credentials* 
++
+Activity parameter containing all Google API credentials. 
+** *Project ID* 
++
+Project ID of the Google API credentials.
+** *Token URI* 
++
+OAuth2 token URI for the Google API credentials.
+** *Universe Domain* 
++
+Universe domain of the Google API credentials.
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
++
+** *File ID*
++
+ID of the file to download or export.
+** *Google drive file type*
++
+Type of Google file to export. Available types are: 
++
+*** *Google Docs*
+*** *Google Sheets*
+*** *Google Slides*
++
+** *Operation mode*
++
+Operation to perform, either *DownloadFile* or *ExportDocument*.
+** *Docs type* 
++
+File format for exporting Google Docs. Available types: 
++
+*** *Microsoft Word (.docx)*
+*** *OpenDocument (.odt)*
+*** *Rich Text (.rtf)*
+*** *PDF (.pdf)*
+*** *Plain Text (.txt)*
+*** *Web Page (HTML) (.zip)*
+*** *EPUB (.epub)*
++
+** *Sheets Type*
++
+File format for exporting Google Sheets. Available types:
++
+*** *Microsoft Excel (.xlsx)*
+*** *OpenDocument (.ods)*
+*** *PDF (.pdf)*
+*** *Web Page (HTML) (.zip)*
+*** *Comma-Separated Values (first-sheet only) (.csv)*
+*** *Tab-Separated Values (first-sheet only) (.tsv)*
++
+** *Slides Type*
++
+File format for exporting Google Slides. Available types:
++
+*** *Microsoft PowerPoint (.pptx)*
+*** *ODP (.odp)* 
+*** *PDF (.pdf)* 
+*** *Plain Text (.txt)*
+*** *JPEG (first-slide only) (.jpg)*
+*** *PNG (first-slide only) (.png)*
+*** *Scalable Vector Graphics (first-slide only) (.svg)*
+
+* *Google Drive File Settings*
+** *Create directory if it does not exist*
++
+When selected, creates the folder specified in *Directory path* if it does not exist.
+** *Directory path* 
++
+Path to the directory where to download or export the file. 
+** *New file name* 
++
+Filename to use when *Set new file name* is enabled. You must specify the file name including the extension. 
+** *Overwrite existing file* 
++
+When selected, overwrites any existing file in the specified download directory that has the same name as the downloaded or exported file.
+** *Set new file name*
++
+When selected, renames the downloaded or exported file using the specified *New file name*.
+
+== Inbound Variables 
+
+* *File ID*
+* *Directory Path* 
+* *Create directory if it does not exist*
+* *Overwrite existing file* 
+* *Set new file name* 
+* *New file name* 
+* *Credential Type*
+* *Project ID* 
+* *Private Key ID* 
+* *Private Key* 
+* *Token URI* 
+* *Auth Provider x509 Cert URL* 
+* *Auth URI* 
+* *Client Email* 
+* *Client ID* 
+* *Client x509 Cert URL* 
+* *Universe Domain* 
+* *Google API Service Account Credentials* 
+
+== Outbound Variables 
+
+* *Download Succeeded* (Boolean)
++
+Confirms if the download was successful. 
+* *Target folder* 
++
+String containing the directory path to the downloaded or exported file. 
+* *File name* 
++
+String containing the file name, including the extension. 
+
+== Wizard 
```

Comment:
I really think this should be a task (imperative) heading describing what the user wants to do. The problem with these single-word generic headings is that search won't catch what the user is looking for - and might return multiple "Wizard" headings.

Download or Export a Google Workspace Document (or something like that)

Reply:
I tried my best to convert these to tasks because users don't actually perform a task here, they configure the action step to perform the tasks when the automation runs.